backend/person-service/src/main/kotlin/com/nosql/personservice/converter/ConfidentialDtoToConfidentialEntityConverter.kt

```diff
@@ -0,0 +1,23 @@
+package com.nosql.personservice.converter
+
+import com.nosql.personservice.dto.ConfidentialDto
+import com.nosql.personservice.entity.ConfidentialEntity
+import org.bson.types.ObjectId
+import org.springframework.core.convert.converter.Converter
+import org.springframework.stereotype.Component
+
+@Component
+class ConfidentialDtoToConfidentialEntityConverter(
+    private val passportDataDtoToPassportDataEntityConverter: PassportDataDtoToPassportDataEntityConverter,
```

Comment:
maybe use the conversion service with lazy initialization here?

Reply:
Fixed, but we can have troubles)))))